core/modules/field_ui/field_ui.admin.inc

```diff
@@ -899,7 +899,12 @@ function field_ui_display_overview($entity_type, $bundle) {
  * @ingroup forms
  */
 function field_ui_display_form($form, &$form_state, $entity_type, $bundle, $view_mode) {
-  $bundle_name = $bundle->name;
+  if (is_object($bundle) && property_exists($bundle, 'name')) {
+    $bundle_name = check_plain($bundle->name);
+  }
+  elseif (is_object($bundle) && property_exists($bundle, 'label')) {
+    $bundle_name = check_plain($bundle->label);
+  }
   $page_title = field_ui_manage_display_title($entity_type, $view_mode);
   backdrop_set_title(t('!page: !teaser display', array(
     '!page' => $bundle_name,
```

Comment:
It's still expecting $bundle_name. Either have a default if the entity doesn't have one or can it be fixed on the multifield side?

Reply:
In this function you'll regardless need to set a default value for $bundle_name. Perhaps right at top of function `$bundle_name = "";` at least, to prevent PHP warnings.

Reply:
Done.